test/check-application

```diff
@@ -35,7 +35,7 @@ class TestApplication(testlib.MachineCase):
 
         # Check Empty State is present if service was stopped when certificates page is open
         m.execute("systemctl stop certmonger.service")
-        b.wait_in_text(".pf-c-empty-state h1", "The certificate service is not active")
```

Comment:
What does it say now?

Reply:
Oh this change which was already merged in #10. The selector was wrong, and i just decided to check for .pf-c-empty-state since it's shorter and does the job.